internal/pkg/cli/svc_deploy.go

```diff
@@ -305,6 +309,17 @@ func (o *deploySvcOpts) configureClients() error {
 		return fmt.Errorf("assuming environment manager role: %w", err)
 	}
 
+	d, err := describe.NewEnvDescriber(describe.NewEnvDescriberConfig{
+		App:         o.appName,
+		Env:         o.envName,
+		ConfigStore: o.store,
+	})
+	if err != nil {
+		return fmt.Errorf("create describer for environment %s in application %s: %w", o.envName, o.appName, err)
+	}
+	o.envDescriber = d
+	o.subnetLister = ec2.New(defaultSessEnvRegion)
```

Comment:
Should this be the session from the EnvManagerRole with the environment's region?

The dream state that we're trying to maintain I think:
* `init` commands use the default session (as it's administrative work to create resources)
* `deploy`, `show`, `status`, ... the rest use the env manager role. This way in the future admins can potentially restrict the EnvManagerRole even further and not allow devs on their teams to take certain operations.

I know that today we are not fully there yet, but we should take small steps towards it 🥾

Reply:
Makes sense! I've modified the `EnvManagerRole` adding a permission. Let me know what you think!